Sprintfinity3902/Content/Floor 1 Room Templates/BossRoom.csv

```diff
@@ -2,8 +2,8 @@ RMEX,,,,,,,,,,,,
 RMIN,,,,,,,,,,,,
  , , , , , , ,BLOK,BLOK,BLOK,BLOK,BLOK, 
  , , , , , , , , ,BLOK,BLOK,BLOK,
- , , , , , , , , , ,BLOK,BLOK,
- , , , , , , , , ,BOSS, , ,
+ , , , , , , , , ,BOSS,BLOK,BLOK,
```

Comment:
Brad and I were just talking about this. I like this fix.

Reply:
Ope, I thought this did something else. Brad and I are worried that the entrance to the boss room might be that right door, which makes the fight kinda awkward for the player.